charts/livy/Chart.yaml

```diff
@@ -15,4 +15,4 @@ sources:
 maintainers:
 - name: jahstreet
   email: jahstreetlove@gmail.com
-kubeVersion: 1.11.0 - 1.18.0
+kubeVersion: 1.11.0 - 1.18.9
```

Comment:
This is still not working with EKS:
`Error: Failed to render chart: exit status 1: Error: chart requires kubeVersion: 1.11.0 - 1.18.9 which is incompatible with Kubernetes v1.18.9-eks-d1db3c`
I would suggest either dropping this entirely or using this (although i can not test if this works on regular k8s):
`kubeVersion: 1.11.0-r0 - 1.18.9-r0`

Reply:
Yeah, seeing the same result. 
@jahstreet : Do we need the `kubeVersion` param?

Reply:
I see lots of questions related to that, though not 100% sure how to properly notify users that there are certain limitations due to the K8s client used in Livy and Spark. I think we can drop kubeVersion here and instead add comment to NOTES.txt. Lemme suggest the change.

Reply:
Things were working for kubernetes 1.21. recently we did an upgrade to 1.22 and since the API version for ingress has changed, there is a failure:
Failure executing: GET at: https://kubernetes.default.svc.cluster.local/apis/extensions/v1beta1/namespaces/.....
It gets 404. 
Any plans for getting the whole livy support with k8 changes merged to official livy repo and receive support on these?

Reply:
Hi @rangnar14 I am facing the same problem. Did you manage to solve it?